lib/atoms/font/makeFontSizes.ts

```diff
@@ -1,74 +1,60 @@
-import merge from 'lodash/merge';
+import { TextDefinition } from './../utils/alignTextToGrid';
 import alignTextToGrid from '../utils/alignTextToGrid';
-import createResponsiveRules from '../utils/createResponsiveRules';
 import { px } from '../utils/toUnit';
-import each from 'lodash/each';
-import { Tokens } from 'lib/themes/theme';
+import makeDesktopRules from '../utils/makeDesktopRules';
+import { Tokens } from '../../themes/theme';
 
-export default (tokens: Tokens) => {
-  const rules: Array<{ [index: string]: object }> = [];
-
-  each(
-    {
-      ...tokens.text,
-      ...tokens.heading
-    },
-    (type, typeName) => {
-      const {
-        fontSize: mobileFontSize,
-        lineHeight: mobileLineHeight
-      } = alignTextToGrid(type.mobile, tokens);
-      const {
-        fontSize: desktopFontSize,
-        lineHeight: desktopLineHeight
-      } = alignTextToGrid(type.desktop, tokens);
+const getFontSizeRules = (textDefinition: TextDefinition, tokens: Tokens) => {
+  const { fontSize, lineHeight } = alignTextToGrid(textDefinition, tokens);
 
-      const mobileRules = {
-        fontSize: mobileFontSize,
-        lineHeight: mobileLineHeight
-      };
-      const desktopRules = {
-        fontSize: desktopFontSize,
-        lineHeight: desktopLineHeight
-      };
-
-      rules.push(
-        createResponsiveRules({
-          tokens,
-          selector: `.fontSize_${typeName}`,
-          mobileRules,
-          desktopRules
-        })
-      );
+  return {
+    fontSize,
+    lineHeight
+  };
+};
 
-      if (typeName === 'standard') {
-        const interactionHeight = tokens.interactionRows * tokens.rowHeight;
-        const mobileInteractionPadding = px(
-          (interactionHeight - type.mobile.rows * tokens.rowHeight) / 2
-        );
-        const desktopInteractionPadding = px(
-          (interactionHeight - type.desktop.rows * tokens.rowHeight) / 2
-        );
+const makeRules = (responsiveType: 'mobile' | 'desktop', tokens: Tokens) => {
```

Comment:
We have a poorly named type called `Breakpoint` that is the type for this key on the reponsive text part of the tokens